iib/web/api_v1.py

```diff
@@ -719,6 +727,7 @@ def regenerate_bundle_batch():
                 build_request.get('organization'),
                 request.id,
                 build_request.get('registry_auths'),
+                build_request.get('bundle_replacements'),
```

Comment:
`handle_regenerate_bundle_request ` will be called with args that it doesn't support. Maybe make this change with the workers? Or you can simply have `handle_regenerate_bundle_request` accept this parameter for now and do nothing with it for the time being. Also, since we don't have any changes in the documentation or the API response, the user will not know about this change which is good.

Reply:
This is part of the reason I wasn't sure why we spliced into the API and the worker. If we are going to continue with that split, it makes more sense to me to fully flesh out the API (and have it tested, but leaving out documentation/API until worker is done), no-oping/ignoring the parameters passed into the worker.

Reply:
Resolving this conversation now that it is irrelevant since I put in worker changes to the same PR.